build-info-extractor-gradle/src/main/groovy/org/jfrog/gradle/plugin/artifactory/ArtifactoryPlugin.groovy

```diff
@@ -24,6 +26,13 @@ class ArtifactoryPlugin extends ArtifactoryPluginBase {
         return result
```

Comment:
Lets fix the above message to:
`Adds artifacts and generates build-info to be later deployed to Artifactory.`